test/widgets/content_test.dart

```diff
@@ -1017,50 +1002,27 @@ void main() {
         });
     });
 
-    testWidgets('maintains font-size ratio with surrounding text, when showing TeX source', (tester) async {
-      const html = '<span class="katex">'
-        '<span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>λ</mi></mrow>'
-          '<annotation encoding="application/x-tex"> \\lambda </annotation></semantics></math></span>'
-        '<span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6944em;"></span><span class="mord mathnormal">λ</span></span></span></span>';
-      await checkFontSizeRatio(tester,
-        targetHtml: html,
-        targetFontSizeFinder: mkTargetFontSizeFinderFromPattern(r'λ'));
-    }, skip: true // TODO(#46): adapt this test
-                  //   (it needs a more complex targetFontSizeFinder;
-                  //    see other uses in this file for examples.)
-    );
-
-    testWidgets('maintains font-size ratio with surrounding text, when showing TeX source', (tester) async {
-      addTearDown(testBinding.reset);
-      final globalSettings = testBinding.globalStore.settings;
-      await globalSettings.setBool(BoolGlobalSetting.renderKatex, false);
-
-      const html = '<span class="katex">'
+    testWidgets('maintains font-size ratio with surrounding text, when fallback to TeX source', (tester) async {
+      const unsupportedHtml = '<span class="katex">'
         '<span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>λ</mi></mrow>'
           '<annotation encoding="application/x-tex"> \\lambda </annotation></semantics></math></span>'
-        '<span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6944em;"></span><span class="mord mathnormal">λ</span></span></span></span>';
+        '<span class="katex-html" aria-hidden="true">'
+          '<span class="base unknown">' // Server doesn't generate this 'unknown' class.
+          '<span class="strut" style="height:0.6944em;"></span>'
+          '<span class="mord mathnormal">λ</span></span></span></span>';
       await checkFontSizeRatio(tester,
-        targetHtml: html,
+        targetHtml: unsupportedHtml,
         targetFontSizeFinder: mkTargetFontSizeFinderFromPattern(r'\lambda'));
     });
 
-    testWidgets('displays KaTeX source; experimental flag disabled', (tester) async {
-      addTearDown(testBinding.reset);
-      final globalSettings = testBinding.globalStore.settings;
-      await globalSettings.setBool(BoolGlobalSetting.renderKatex, false);
-
+    testWidgets('displays KaTeX content', (tester) async {
       await prepareContent(tester, plainContent(ContentExample.mathInline.html));
-      tester.widget(find.text(r'\lambda', findRichText: true));
+      tester.widget(find.text('λ', findRichText: true));
     });
 
-    testWidgets('displays KaTeX content; experimental flag enabled', (tester) async {
```

Comment:
nit: make summary line more specific

    content test [nfc]: Reorder KaTeX widget tests
    
    With renderKatex flag enabled (and soon to be default behaviour)
    being first.

There's room in that summary line to put some of that information from the body, to give an idea of what the reordering is about. Like:
> content test [nfc]: Reorder KaTeX widget tests, enabled before disabled